### ISSUE TITLE
Add support for Discourse

### ISSUE THREAD

**Issue by openscript** [Sommerregen/grav-plugin-jscomments#2]
Discourse can also be used as a page threaded commenting system. It's easy to [embed](https://meta.discourse.org/t/embedding-discourse-comments-via-javascript/31963) into pages by inserting some JavaScript, very similar to Discus.

I'll probably implement it in the next few days.

**Comment by Sommerregen** [comment]
Hi @openscript,

thank you for the addition! In principle you can use [disqus.html.twig](https://github.com/Sommerregen/grav-plugin-jscomments/blob/master/templates/plugins/jscomments/disqus.html.twig) as a template. According to the docs there are only few modifications needed. Additional you just have to add a config entry in the `blueprints.yaml` like https://github.com/Sommerregen/grav-plugin-jscomments/blob/master/blueprints.yaml#L29-L39 .

Would be nice to see it as a PR. However, it's on my roadmap now for the next version.

**Comment by openscript** [comment]
I'll send you a PR until 14.05.16. I hope you didn't start working on it yet, so we don't implement it twice :collision:

**Comment by Sommerregen** [comment]
Hi @openscript ,

that's great! :+1:   I'll wait for your PR.

**Comment by openscript** [comment]
It was pretty easy to integrate Discourse. I verified if it's working by setting up a local installation of Discourse and a new Grav instance. 

![image](https://cloud.githubusercontent.com/assets/1105080/15213458/ae26e3b4-1847-11e6-8bc3-db7328b53e58.png)

I'm not sure, if it's okay to increase the version number as I did, but I think it's a bigger change to add support for something so I incremented the minor number.

Let me know, if I can do anything more. Thank you for creating this plugin :+1:

**Comment by Sommerregen** [comment]
Ah, cool! Never mind about version increment. I was refactoring the plugin nevertheless and added lots of stuff (a nicer admin plugin settings page for example). I merged your PR #3 and made some minor adjustments. One adjustments is the naming; instead of `url` I'm using `host` e.g.,

``` yaml
providers:
  discourse:
    host: ''                # The URL to the Discourse commenting system
```

and updated your Twig template of course. I hope you don't mind. Other than testing there is nothing to do at the moment. The changes should popup in an hour via GPM provided you are on the "testing" channel.

Thanks again!

**Comment by openscript** [comment]
Perfect! Thank you too! :-)
